pyanaconda/exception.py

```diff
@@ -138,6 +138,10 @@ def handleException(self, dump_info):
         """
 
         log.debug("running handleException")
+        # don't try and attach empty or non-existent files (#2185827)
+        self.conf.fileList = [
+            fn for fn in self.conf.fileList if os.path.exists(fn) and os.path.getsize(fn) > 0
```

Comment:
`os.path.isfile(fn)` might be even a bit safer than `.exists()`. But that's just a nitpick.